accessibility/accessibility.go

```diff
@@ -35,16 +35,19 @@ func PromptInt(w io.Writer, r io.Reader, prompt string, low, high int) int {
 	return choice
 }
 
-func parseBool(s string) (bool, error) {
-	s = strings.ToLower(s)
+func parseBool(s string, val bool) (bool, error) {
+	s = strings.TrimSpace(strings.ToLower(s))
+	if s == "" {
+		return val, nil
+	}
```

Comment:
I'd rename `val` to something like `defaultValue`:

```suggestion
func parseBool(s string, defaultValue bool) (bool, error) {
	s = strings.TrimSpace(strings.ToLower(s))
	if s == "" {
		return defaultValue, nil
	}
```